.github/workflows/deploy.yml

```diff
@@ -0,0 +1,50 @@
+# yaml-language-server: $schema=https://json.schemastore.org/github-workflow.json
+name: Deploy to Vercel CI
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [main,develop]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    if: "!contains(github.event.head_commit.message, '[skip ci]')"
```

Comment:
Is this `if` condition still needed, now that we manually trigger the deployment?

Reply:
it could be used if we attach `[skip ci]` to the merge commit i guess – if we don't want to update staging/production. don't know how often this will be the case though.

Reply:
fair enough, doesn't hurt either to have that option